mastering-plone/volto_components_sponsors.md

```diff
@@ -264,7 +263,7 @@ now render the sponsors data. As we have already prepared a dictionary by sponso
 </List>
 ```
 
-````{admonition} Complete code of the `Sponsors` component
+:::{admonition} Complete code of the Sponsors component
```

Comment:
Use 4 backticks here and match below
```suggestion
````{admonition} Complete code of the Sponsors component
```

Also is it literally `admonition`, and not `info` or `warning` or some other admonition?

Reply:
I would prefer to avoid using admonition, but it is the one that allows a title.

:::{admonition} is a relict. It is useful, when the title includes a backtick to avoid error / warning.
But as I removed the backticks, the ````{admonition} will do.

Reply:
I wonder if MyST can pass four colons as the parent, so you can use backticks in the title?

Also thanks for teaching me about that admonition feature. https://docutils.sourceforge.io/docs/ref/rst/directives.html#generic-admonition